src/Extensions/Shopware/Install/Services/Install/Release.php

```diff
@@ -108,28 +108,31 @@ public function __construct(
      */
     public function installShopware(InstallationRequest $request)
     {
-        $this->releaseDownloader->downloadRelease($request->getRelease(), $request->getInstallDir());
-
-        if ($request->getRelease() === 'latest' || version_compare($request->getRelease(), '5.1.2', '>=')) {
-            $this->createDatabase($request);
-            $this->createShopwareConfig($request);
-            $this->runInstaller($request);
-        } else {
-            $this->generateVcsMapping($request->getAbsoluteInstallDir());
-            $this->createShopwareConfig($request);
-            $this->setupDatabase($request);
-            $this->lockInstaller($request->getAbsoluteInstallDir());
+        if (!$request->getSkipDownload()) {
+            $this->releaseDownloader->downloadRelease($request->getRelease(), $request->getInstallDir());
         }
+        if (!$request->getOnlyUnpack()) {
```

Comment:
Can you implement it like this please? It makes it a little bit easier to read and it is not a negative check ;)

```
if ($request->getOnlyUnpack()) {
    return;
}

if ($request->getRelease() === 'latest' || ...
```